test/e2e/magellan-calypso.json

```diff
@@ -1,9 +1,8 @@
 {
-	"mocha_tests": [ "specs" ],
-	"mocha_args": "-R spec-junit-reporter",
+	"mocha_tests": [ "specs/specs-calypso" ],
 	"max_test_attempts": 3,
-	"max_workers": 6,
-	"suiteTag": "parallel;signup",
+	"max_workers": 1,
```

Comment:
This should be the default when running tests locally. In that scenario, we probably don't want to run multiple chrome instances.

This is overridden by `--max_workers=%E2E_WORKERS%` when running tests in TeamCity.